telegram/ext/_dispatcher.py

```diff
@@ -643,10 +746,10 @@ def update_persistence(self, update: object = None) -> None:
 
     def __update_persistence(self, update: object = None) -> None:
         if self.persistence:
-            # We use list() here in order to decouple chat_ids from self.chat_data, as dict view
+            # We use list() here in order to decouple chat_ids from self._chat_data, as dict view
             # objects will change, when the dict does and we want to loop over chat_ids
-            chat_ids = list(self.chat_data.keys())
-            user_ids = list(self.user_data.keys())
+            chat_ids = list(self._chat_data.keys())
+            user_ids = list(self._user_data.keys())
```

Comment:
why user the privates  here? I don't see the need for it and the less we use them, the safer we are